app/Http/Controllers/NovaExportController.php

```
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\Storage;
+use League\Flysystem\FileNotFoundException;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+
+class NovaExportController extends Controller
+{
+    /**
+     * @param string $file
+     *
+     * @return JsonResponse|BinaryFileResponse
+     */
+    public function export(string $file)
+    {
+        try {
```

Tbh I would just let this bubble up whatever exceptions are thrown.